src/Types.hs

```diff
@@ -481,8 +481,10 @@ hasUnread' :: ClientChannel -> Bool
 hasUnread' chan = fromMaybe False $ do
     let info = _ccInfo chan
     lastViewTime <- _cdViewed info
-    return $ ((_cdUpdated info) > lastViewTime) ||
-             (isJust $ _cdEditedMessageThreshold info)
+    return $ _cdMentionCount info > 0 ||
```

Comment:
I'm curious why the `_cdMentionCount` check occurs even if the channel is muted. Is that how the web client behaves?

Reply:
The web client treats mentions in a muted channel as unread messages. If you use keyboard shortcuts to jump to the next channel with unread messages, this includes muted channels with mentions. These channels are displayed the same way as channels with mentions normally are, except darker. Here's what this looks like for me (testing999 is muted, but Town Square isn't):
<img width="219" alt="Screen Shot 2020-07-08 at 9 51 44 AM" src="https://user-images.githubusercontent.com/14264467/86947481-ade75080-c100-11ea-9cad-abf41d45d78f.png">

In Matterhorn on this branch, testing999 is highlighted as if it has normal unread messages, but the mention count is still displayed.
<img width="148" alt="Screen Shot 2020-07-08 at 9 55 19 AM" src="https://user-images.githubusercontent.com/14264467/86947893-467dd080-c101-11ea-9c07-378d076fd838.png">

Reply:
Okay, thanks!